src/gui/hud/SecondaryInventory.cpp

```diff
@@ -558,7 +558,8 @@ bool SecondaryInventoryHud::dragEntity(Entity * io, const Vec2s & pos) {
 		}
 	}
 	
-	return false;
+	Set_DragInter(io);
```

Comment:
The thing is, they weren't there before. `Set_DragInter` felt logical, but I wasn't too sure about the others. I figure `ARX_INVENTORY_IdentifyIO` would attempt to identify the item. So yeah, that should be there. But what purpose does `RemoveFromAllInventories` serve?